src/App.test.js

```diff
@@ -1,8 +1,54 @@
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import userEvent from '@testing-library/user-event';
+import App from './app';
 
-test('renders learn react link', () => {
+window.scroll = jest.fn();
+
+test('renders the Home page', () => {
+  render(<App />);
+  expect(
+    screen.getAllByText(/Welcome to the Codecademy Discord Community/i)[0]
```

Comment:
This could be a `getByText` since there is only place where this text is on the page. We also don't need the case-insensitive flag on all the regex in these tests.  Let's just test for exact matches

Reply:
Ok. I made the change.